### ISSUE TITLE
Consider adding `Version` in the schema

### ISSUE THREAD

**Issue by vincentsarago** [opengeospatial/2D-Tile-Matrix-Set#82]
Hi, 
First thanks for the hard work put in defining the specificatio. It's true that I usually mostly complain about OGC specs because I find them hard to follow. Anyway I wanted to propose we add a `Version` into the TMS specification (maybe it exists but I didn't find it), because right now it's pretty hard to see in which version a TMS is based on. IMO having a simple `version: 1.0` or `version: 2.0` will make it easier. 

I understand this comes late to the party but if this is something that could be considered for later version it will be great IMO.

Thank you

**Comment by jerstlouis** [comment]
Thank you very much for the feedback @vincentsarago , I agree this would have been an idea, and we should do this for later versions.

In truth, I don't think we really expected the TileMatrixSet standard to get much used used before we (OGC) put out a standard (e.g., _OGC API - Tiles_) that referenced it, but I realized that projects like _TiTiler_ actually used it :).

For reading both version 1.0 and 2.0, I think the easiest approach would be to support both the old and new name for the properties.
